Components/Badges/badges.css

```diff
@@ -0,0 +1,68 @@
+@import url('../../styles/utility.css');
+@import url('../../styles/navBar.css');
+@import url('../../styles/sideBar.css');
+@import url('../../styles/variable.css');
+@import url('../Avatar/avatar.css');
+
```

Comment:
suggestion You make a separate file where you keep all the imported external sheets (main.css) then use this main.css in doc.css.

Reply:
thankou for suggestion